ui.apps/src/main/content/jcr_root/apps/custom_submit_action/.content.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@rakeshkumar-verma : is the custom_submit_action sling folder part of forms setup ?

Reply:
@rishabh-behl  yes